src/GLM.jl

```diff
@@ -14,10 +14,11 @@ import Statistics: cor
 using StatsAPI
 import StatsBase: coef, coeftable, coefnames, confint, deviance, nulldeviance, dof,
                   dof_residual,
-                  loglikelihood, nullloglikelihood, nobs, stderror, vcov,
+                  leverage, loglikelihood, nullloglikelihood, nobs, stderror, vcov,
                   residuals, predict, predict!,
                   fitted, fit, model_response, response, modelmatrix, r2, r², adjr2, adjr²,
-                  PValue
+                  PValue,
+                  fweights, pweights, aweights
```

Comment:
Spotted by @devmotion at #619: these don't need to be imported actually, better move them to the `using StatsBase:...` line above.